src/data/init/init-abilities.ts

```diff
@@ -202,6 +203,12 @@ import { TerrainType } from "#enums/terrain-type";
 import { WeatherType } from "#enums/weather-type";
 import i18next from "i18next";
 
+/** Used for Aerialate, Refrigerate, Pixilate, Galvanize */
+const basicMoveTypeChangeAbilityCondition: PokemonAttackCondition = (user, _target, move) =>
+  move?.type === ElementalType.NORMAL
+  && (!move.hasAttr(VariableMoveTypeAttr)
+    || ([MoveId.TERA_BLAST, MoveId.TERA_STARSTORM].includes(move.id) && !user?.isTerastallized));
```

Comment:
NIT: Please create a (frozen) array for those. e.g. in `move-utils`

```ts
const teraMoveIds = Object.freeze([MoveId.TERA_BLAST, MoveId.TERA_SANDSTORM]);
```

and then
```suggestion
    || (teraMoveIds.includes(move.id) && !user?.isTerastallized));
```

----

ALT: Create a helper method `move.isTeraMove()` (it could still check that array)